index.js

```diff
@@ -19,7 +19,8 @@ module.exports = function(source) {
 
     const callback = this.async(),
         options = Object.assign({}, this.options.bemLoader, loaderUtils.getOptions(this)),
-        levelsMap = options.levels || bemConfig.levelMapSync(),
+        levelsFromSets = options.sets && bemConfig.levelsSync(options.sets).map(level => level.path),
+        levelsMap = levelsFromSets || bemConfig.levelMapSync() || options.levels,
```

Comment:
Hmm. Why do we use `levelsSync().map()` above? Is `levelMapSync` doing same thing?

Reply:
This is all about strange method names... ok